pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "dcicwrangling"
+version = "0.1.0"
+description = "Scripts and Jupyter notebooks for 4DN wrangling"
+authors = ["4DN-DCIC Team <support@4dnucleome.org>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/4dn-dcic/dcicwrangling"
+repository = "https://github.com/4dn-dcic/dcicwrangling"
+packages = [{ include="scripts", from="." }]
+
+[tool.poetry.dependencies]
+python = ">=3.6, <3.8"
```

Comment:
As stated previously, if you don't support 3.7 just explicitly disallow it here. I'm guessing it is not much work to be compatible, but not sure.